### PR TITLE
Add Amman to docs.

### PR DESCRIPTION
**Pull request by tonyboylehub**
Migrated across Amman docs and added some snippets.

### CODE REVIEW

**Comment by blockiosaurus** [comment]
Is this a typo in Amman?
killlRunningRelay. I saw it in the premade configs.

File: src/pages/amman/configuration.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+metaTitle: Amman - Configuration
+description: Configuring Amman local validator toolkit.
+---
+
+When executed Amman will look for the configuration `.ammanrc.js` in the project root. If this file is not present Amman will load up with a default configuration.
+
+The config should be a JavaScript module exporting 'validator' with any of the below
+properties:
+
+- killRunningValidators: if true will kill any solana-test-validators currently running on the system.
```
Comment (blockiosaurus) [inline src/pages/amman/configuration.md:12]: Nit: can you bold these Args so they stand out,m

**Review by blockiosaurus — COMMENTED** [review]
This looks really good! Just had a nit and a question and then good to go!

**Comment by tonyboylehub** [comment]
> Is this a typo in Amman? killlRunningRelay. I saw it in the premade configs.

Looks like it's a type from the original docs on github.

File: src/pages/amman/configuration.md (hunk 1 above)
Comment (tonyboylehub) [reply src/pages/amman/configuration.md]: Added.

**Review by blockiosaurus — APPROVED** [review]

**Pull request merged**